packages/main/src/TabContainer.hbs

```diff
@@ -3,7 +3,7 @@
 	dir="{{rtl}}"
 >
 	<div class="{{classes.header}}" id="{{_id}}-header">
-		<ui5-icon @click="{{_onHeaderBackArrowClick}}" class="{{classes.headerBackArrow}}" src="sap-icon://slim-arrow-left" tabindex="-1"></ui5-icon>
+		<ui5-icon @click="{{_onHeaderBackArrowClick}}" class="{{classes.headerBackArrow}}" src="sap-icon://slim-arrow-left" tabindex="-1" accessible-name="Previous" icon-tooltip="true"></ui5-icon>
```

Comment:
We have to ensure that the "Previous" and "Next" texts would be translated. In ui5-webcomponents we have similar translation process as in openui5.

(1) You have to add those texts in the `packages/main/src/18n/messagebundle.properties` file in first place.
(every week on Wednesday a translation team delivers translations for new texts )

(2) You have to use the `i18nBundle` in `TabContainer.js`
import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";

(3) Get the text like this:

```js
 // TabContainer.js
get nextIconACCName() {
     return this.i18nBundle.getText(TABCONTAINER_NEXT_ICON_ACC_NAME);
 }
```

(4) And use it in the .hbs file:

```html
<ui5-icon  accessible-name="{nextIconACCName}" />
```

You can see how to use the `i18nBundle` in the `Badge.js`, so use it for reference.
In case you need more information how to do it, feel free to contact us.

Reply:
This is also applicable to the title="Overflow Menu"